### DISCUSSION TITLE
posts/javase/001.introduce

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#360] [Announcements]
# posts/javase/001.introduce

IT蚱蜢哥|Java|大数据|Oracle|HIVE|java,java教程,Java,mysql,rabbitmq,rabbitmq教程,javascript,设计模式,GOF23设计模式,编程,开发,教程,软件开发,互联网,spring,程序员,程序

https://itzhameng.com/posts/javase/001.introduce.html

**Comment by itzhameng** [comment]
Java基础知识

**Comment by itzhameng** [comment]
test

**Reply by Mister-Hope** [reply]
I have to warn you that I made a notice in template comment that using this repo in production is NOT  ALLOWED.
